src/style/app.css

```diff
@@ -1,4 +1,8 @@
+@import 'tailwindcss';
 @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,500;0,600;0,700;0,900;1,400;1,500');
+
+@config '../../tailwind.config.js';
```

Comment:
This is just a complaint and not a knock on this PR - I wonder why we have this big app.css file and also tailwind.css ... I'm no longer the fan of Tailwind I used to be.